hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -630,8 +629,7 @@ private void migrateSchemas(
         if (trigger == GENESIS) {
             final var config = configProvider.getConfiguration();
             final var ledgerConfig = config.getConfigData(LedgerConfig.class);
-            final var genesisRoster = buildRoster(requireNonNull(genesisAddressBook));
-            genesisNetworkInfo = new GenesisNetworkInfo(genesisRoster, ledgerConfig.id());
+            genesisNetworkInfo = new GenesisNetworkInfo(requireNonNull(genesisAddressBook), ledgerConfig.id());
```

Comment:
It was intentional to use Roster inside `GenesisNetworkInfo`. How does it matter if we are using addressBook or Roster here ?

Reply:
The swapping of the order of the ip addresses in the roster vs the hedera address book was how I broke integration two weeks ago.  They were not being swapped when going from roster -> genesis network info.  

The roster was always intended to be derived from Hedera Address Book and not used to define the Hedera Address Book.   The situation will be moot once we adopt the new format for genesis network description `genesis_network.json`.   The roster just doesn't have enough information to properly define the Hedera AB.